receiver/elasticapmreceiver/internal/mappers/IntakeV2ToSemConv.go

```diff
@@ -0,0 +1,83 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// This file contains all the mapping from IntakeV2 fields to OTel Semantic Convention
+
+package mappers
+
+import (
+	"github.com/elastic/apm-data/model/modelpb"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv "go.opentelemetry.io/collector/semconv/v1.5.0"
+)
+
+// Translates resource attributes from the Elastic APM model to the OpenTelemetry resource attributes
+func TranslateToOtelResourceAttributes(event *modelpb.APMEvent, attributes pcommon.Map) {
+	attributes.PutStr(semconv.AttributeServiceName, event.Service.Name)
+	attributes.PutStr(semconv.AttributeServiceVersion, event.Service.Version)
+	attributes.PutStr(semconv.AttributeTelemetrySDKLanguage, event.Service.Language.Name)
+	attributes.PutStr(semconv.AttributeTelemetrySDKName, "ElasticAPM")
+	attributes.PutStr(semconv.AttributeDeploymentEnvironment, event.Service.Environment)
+}
+
+func TranslateIntakeV2TransactionToOTelAttributes(event *modelpb.APMEvent, attributes pcommon.Map) {
+	if event.Http.Request != nil {
+		attributes.PutStr(semconv.AttributeHTTPMethod, event.Http.Request.Method)
+		attributes.PutInt(semconv.AttributeHTTPStatusCode, int64(event.Http.Response.StatusCode))
+
+		if event.Url != nil {
+			attributes.PutStr(semconv.AttributeHTTPURL, event.Url.Full)
```

Comment:
we need the new SemConv for URL here, NOT `http.url` (that is deprecated):
- url.full
- url.scheme
- url.domain
- ...

see https://github.com/elastic/opentelemetry-lib/blob/995b8f40588aea9477b3c5205704ed6f3aa89f48/enrichments/trace/internal/elastic/span.go#L139

Reply:
Moved to SemConv v1.27.0 - that's the latest version in the latest go package.